libtelemetry/src/test/java/com/mapbox/android/telemetry/SchemaTest.java

```diff
@@ -0,0 +1,528 @@
+package com.mapbox.android.telemetry;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.annotations.SerializedName;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.zip.GZIPInputStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class SchemaTest {
+  private static final String APP_USER_TURNSTILE = "appUserTurnstile";
+  private static final String LOCATION = "location";
+  private static final String MAP_CLICK = "map.click";
+  private static final String MAP_DRAG = "map.dragend";
+  private static final String MAP_LOAD = "map.load";
+  private static final String NAVIGATION_ARRIVE = "navigation.arrive";
+  private static final String NAVIGATION_CANCEL = "navigation.cancel";
+  private static final String NAVIGATION_DEPART = "navigation.depart";
+  private static final String NAVIGATION_FASTER_ROUTE = "navigation.fasterRoute";
+  private static final String NAVIGATION_FEEDBACK = "navigation.feedback";
+  private static final String NAVIGATION_REROUTE = "navigation.reroute";
+  private ArrayList<JsonObject> schemaArray;
+
+  @Before
+  public void downloadSchema() throws Exception {
+    unpackSchemas();
```

Comment:
We're still unzipping the schema on every test which is not necessary 👀 https://github.com/mapbox/mapbox-events-android/pull/160#discussion_r202411192

Although I'd personally rather creating a new task as suggested ☝️ we could also use the [`@BeforeClass`](http://junit.sourceforge.net/javadoc/org/junit/BeforeClass.html) annotation here 👀 

> Annotating a `public static void` no-arg method with `@BeforeClass` causes it to be run once before any of the test methods in the class.